src/h3/mod.rs

```diff
@@ -604,11 +604,22 @@ pub enum Event {
     GoAway,
 }
 
+/// An HTTP/3 setting.
+#[derive(Clone, Debug, PartialEq)]
+pub struct RawSetting {
```

Comment:
Why not just use `(u64, u64)` tuples? Would save us from having to expose another type.